chromiumcontent/chromiumcontent.gypi

```diff
@@ -122,7 +122,7 @@
           },
         },
       }],
-      ['_target_name in ["gtk2ui", "devtools_http_handler", "devtools_discovery", "http_server"]', {
+      ['_target_name in ["gtk2ui", "devtools_http_handler", "devtools_discovery", "http_server", "desktop_capture", "webrtc_common", "system_wrapper", "rtc_base", "rtc_base_approved", "libyuv"]', {
```

Comment:
After a deep look, the code in [chromiumcontent.gypi#L93-L95](https://github.com/atom/libchromiumcontent/blob/master/chromiumcontent/chromiumcontent.gypi#L93-L95) will not be triggered in Debug build(with `component=shared_library`). I think it's fine to remove the `component='static_library'` check for `Debug`/`Release` build there.